packages/react/src/components/carbon-components-react/UIShell/HeaderMenu.js

```diff
@@ -140,15 +140,9 @@ class HeaderMenu extends React.Component {
   handleOnBlur = event => {
     if (!event.currentTarget.contains(event.relatedTarget)) {
       this.setState({ expanded: false, selectedIndex: null });
-      root.document.body.classList.remove(`${prefix}--body__lock-scroll`);
+      root.document?.body?.classList.remove(`${prefix}--body__lock-scroll`);
     }
 
-    const megamenuItems = [
```

Comment:
FYI, I noticed that this was not being used at all so I removed this @annawen1

Reply:
I see, a side note @jeffchew what do you think of enabling [ESLint `no-unused-vars` rule](https://eslint.org/docs/rules/no-unused-vars)?

Reply:
yep, thanks for catching that!